js/src/chrome/settings/styler.js

```diff
@@ -1249,16 +1236,11 @@ Wu.Styler = Wu.Class.extend({
 		return n;
 	},
 
-	_validateDateFormat : function (key) {
-		return Wu.Tools.validateDateFormat(key);
-	},
-
 	_gradientStyle : function (colorArray) {
-		var gradientStyle = 'background: -webkit-linear-gradient(left, ' + colorArray.join() + ');';
```

Comment:
Depends on browser and OS. But point is, it's not much difference.
![screen shot 2016-03-22 at 2 50 14 pm](https://cloud.githubusercontent.com/assets/2197944/13953710/8d5d67d4-f03d-11e5-8fa1-c0e6c0f78d5e.png)